### PR TITLE
Record deployments in k8s.

### PR DESCRIPTION
**Pull request by akatsoulas**
There are some changes coming from the linter. The important part is the `revisionHistoryLimit` and the `--record` flag for the web pods in the kubectl

### CODE REVIEW

File: k8s/deploy_utils.py [hunk 1]
```diff
@@ -30,19 +30,22 @@ def render_template(config, template_name, app):
...
-        f.write("\n".encode('utf-8'))
+    namespace = ctx.config["kubernetes"]["namespace"]
+    with tempfile.NamedTemporaryFile(prefix="k8s", suffix=".yaml", delete=False) as f:
+        f.write(template_text.encode("utf-8"))
+        f.write("\n".encode("utf-8"))
         f.flush()
 
     print("Rendering template to:", f.name)
-    cmd = '{} apply -n {} -f {}'.format(get_kubectl(), namespace, f.name)
+    if record:
```
Comment (smarnach) [inline k8s/deploy_utils.py:45]: Where does this variable come from? Did you mean to add it as a parameter to this funciton?

**Review by smarnach — COMMENTED** [review]
Is there a reason we can't add `--record` unconditionally for all deployments?

File: k8s/deploy_utils.py (hunk 1 above)
Comment (akatsoulas) [reply k8s/deploy_utils.py:45]: Hm, my bad. Thanks for catching this!

**Comment by akatsoulas** [comment]
No reason at all! Celery and cron run the same image so we do have the image deployed from the web pods. But we should be able to rollback everything if needed. I am amending the PR to include it everywhere.

**Comment by akatsoulas** [comment]
@smarnach PR updated

**Review by smarnach — APPROVED** [review]
Looks good to me.

**Pull request merged**